lib/notifiers.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'notifiers/base'
+require_relative 'notifiers/slack'
+require_relative 'notifiers/discourse'
+
+module Notifiers
```

Comment:
Za mna to schovajme do `Uvobot` vsetko a notifikacie do `Uvobot::Notifications` a tam `Uvobot::Notifications::SlackNotifier`.